variables.tf

```diff
@@ -0,0 +1,10 @@
+# variable "name" {
+#   description = "Moniker to apply to all resources in the module"
+#   type        = string
+# }
+
+# variable "tags" {
+#   default     = {}
+#   description = "User-Defined tags"
+#   type        = map(string)
+# }
```

Comment:
delete?

Reply:
I'm gonna be working with those on my next PR so I didn't want them totally gone, just not throwing the "declared but not used" error